src/TMap.cpp

```diff
@@ -49,7 +50,9 @@ TMap::TMap(Host* pH, const QString& profileName)
 , m2DPanXStart(0.0f)
 , m2DPanYStart(0.0f)
 , mTargetID(0)
+#if defined(INCLUDE_3DMAPPER)
 , mpM(Q_NULLPTR)
```

Comment:
:thinking: Actually it might be cleaner code to leave this as a `nullptr` (remember we are phasing out `Q_NULLPTR` BTW) and have the places where the pointer would be dereferenced to check and abort if it is still a null...

Reply:
Might be, but this way works too.